actors/miner/src/lib.rs

```diff
@@ -2995,6 +2996,297 @@ impl Actor {
         Ok(())
     }
 
+    fn move_partitions(
+        rt: &impl Runtime,
+        mut params: MovePartitionsParams,
+    ) -> Result<(), ActorError> {
+        let policy = rt.policy();
+        if params.from_deadline == params.to_deadline {
+            return Err(actor_error!(illegal_argument, "from_deadline == to_deadline"));
+        }
```

Comment:
This could be deleted if we fail it when !deadline_available_for_move earlier (before the transaction handling).

Reply:
same as reply for `deadline_available_for_move`